scripts/run_dummy_models_on_val.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import os
+from src.dummy_models import (
+    dummy_random,
+    dummy_always_bald,
+    dummy_always_non_bald,
+    evaluate_dummy_model,
+)
+
+val_csv_path = os.path.join("..", "src", "data", "val.csv")
+val_df = pd.read_csv(val_csv_path)
+val_labels = val_df["label"].values
+
+output_dir = os.path.join("..", "results")
+os.makedirs(output_dir, exist_ok=True)
+output_file = os.path.join(output_dir, "dummy_model_on_val_metrics_report.txt")
+
+with open(output_file, "w") as report_file:
```

Comment:
first calculate what you nedd, than open a file and write.

Extract saving the results to  function from utils and use here and in training pipeline

Reply:
ok